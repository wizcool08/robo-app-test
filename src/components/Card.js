import React from "react";

const Card = ({ id, name, email }) => {
  return (
    <div className="bg-yellow dib br3 pa3 ma2 shadow-5 bw2 grow">
      <img
        src={`https://robohash.org/${id}`}
        width="200px"
        height="200px"
        alt="robots"
      />
      <div>
        <h2>{name}</h2>
        <p>{email}</p>
      </div>
    </div>
  );
};

export default Card;
