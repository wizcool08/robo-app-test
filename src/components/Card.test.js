import { shallow } from "enzyme";
import React from "react";
import Card from "./Card";

it("render the Card Component", () => {
  expect(shallow(<Card />).debug()).toMatchSnapshot();
});
