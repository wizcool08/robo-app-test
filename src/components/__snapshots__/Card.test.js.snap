// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the Card Component 1`] = `
"<div className=\\"bg-yellow dib br3 pa3 ma2 shadow-5 bw2 grow\\">
  <img src=\\"https://robohash.org/undefined\\" width=\\"200px\\" height=\\"200px\\" alt=\\"robots\\" />
  <div>
    <h2 />
    <p />
  </div>
</div>"
`;
